src/main/java/io/github/jhipster/online/web/rest/AccountResource.java

```diff
@@ -186,19 +185,20 @@ public void changePassword(@RequestBody PasswordChangeDTO passwordChangeDto) {
      */
     @PostMapping(path = "/account/reset-password/init")
     public void requestPasswordReset(@RequestBody String mail) {
-        if (mailService.isEnabled()) {
```

Comment:
where do we handle it now ?

Reply:
Please [see my below comment](https://github.com/jhipster/jhipster-online/pull/175#issuecomment-640106599) and please do correct me if I am wrong. :smile: